docs/user-guide/overview.md

```diff
@@ -10,10 +10,23 @@
 >In the tenant menu you will have access to all the tenants related to the user and you can also quickly switch between one tenant and another.
 ><div align=center><img width="800" src="/assets/images/top-menu/tenant-menu.jpg"/></div>
 
+## Tenants
+>A tenant is like a sales channel. Let's say you have a bussines A where you sale some kind of products so you are going to have a tenant with some connected integrations to manage your products/orders etc. Now, let's say you have another bussines B. For that you can create another tenant and create there connected integrations to manage others product/orders. 
+>In addition, each tenant is a different environment, so they will need to set the environment for each tenant (like install available integrations, create/authorize connected integrations, subscribe to a plan, etc).
+
+>To view the tenants associated with your user, you can go to the tenants menu at the top where the tenants will be listed or if you want a more detailed view you can go to the Settings / Tenants option on the left side menu.
+
+>In this last view you can see the name of the tenant, if it is ready or not to work with the application, expiration date and when it was created and updated. In addition, you will have access to a set of actions such as adding a new tenant, searching for a tenant in the list, updating the list, and removing a tenant. Note that the tenant the user is currently in cannot be deleted, to do so, you will have to switch to a different tenant.
```

Comment:
"Note that the tenant the user is currently in cannot be deleted" to "Note that the tenant where the user is currently active cannot be deleted"